data_sender.py

```diff
@@ -7,11 +7,11 @@
 
 class DataSender:
 
-    CAMERA_TILT = 9999  # update with the correct camera tilt angle
+    CAMERA_TILT = 30  # update with the correct camera tilt angle
 
     def __init__(self):
-        name = "ttyTHS2"
-        rate = 9600  # update with the correct baudrate
+        name = "ttyS0"
```

Comment:
We may want to make this an argument to pass in. This may make it easier to switch different serial sources.